dev_guide/jobs.adoc

```diff
@@ -56,14 +56,32 @@ spec:
       - name: pi
         image: perl
         command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-      restartPolicy: Never
+      restartPolicy: OnFailure    <4>
 ----
 
 1. Optional value for how many pod replicas a job should run in parallel; defaults to `*completions*`.
 2. Optional value for how many successful pod completions are needed to mark a job completed; defaults to one.
 3. Template for the pod the controller creates.
+4. The restart policy of the pod. This does not apply to the job controller. See xref:creating-a-job-known-issues[] for details.
 ====
 
+Alternatively, you can create and launch a job from a single command -- specifically, `oc run`. The following command creates and launches the same job as the `pi` resource:
+
+----
+$ oc run pi --image=perl --replicas=1  --restart=OnFailure --command -- perl -Mbignum=bpi -wle 'print bpi(2000)' 
+----
+
+[[creating-a-job-known-issues]]
+=== Known Limitations
+
+The job specification restart policy only applies to _pods_, not the _job controller_. However, the job controller is hard-coded to keep retrying jobs to completion. 
```

Comment:
s/The job specification restart policy only applies to _pods_, not the _job controller_. /The job specification restart policy only applies to the _pods_, and not the _job controller_. (the,and)